docker/centos7/Dockerfile

```diff
@@ -16,10 +16,11 @@ RUN yum install -y krb5-devel
 ENV NPM_CONFIG_LOGLEVEL info
 RUN git clone https://github.com/tj/n /n
 RUN cd /n && make install
-run n 6.17.1
-run n 8.16.1
-run n 10.16.3
-run n 12.10.0
+RUN n 6.17.1
+RUN n 8.16.1
+RUN n 10.16.3
+RUN n 12.10.0
+RUN n 16.9.0
```

Comment:
2 comments:
1. It would be better to simply call `n 16`, this way we always get the latest `minor.patch` version.
2. We can remove support for versions 6, 8 and 10, maybe 12 as well, and introduce 14

Reply:
Removed minor versions in dockerfile node installations AND in run.sh files.

Also fixed an issue with centos7 build chain.

Compiling with node 16 requires the option '--std=c++14' but this option has been added in gcc version 4.9, default version in centos 7 is 4.8.5.
Now installing gcc 10.

In ArchLinux, it's compiled using gcc 11.1 and in Ubuntu, using gcc 5.3.